spiders.html

```diff
@@ -153,14 +152,15 @@
         scrollY: 700,
         scrollX: true,
         scrollCollapse: true,
+        fixedColumns: true,
```

Comment:
100px horizontal margin is much too big with this option on.

![deploy-preview-20--alltheplaces netlify app_spiders html(Pixel 5) (1)](https://user-images.githubusercontent.com/23022/185915806-54f30ad1-bd3e-4873-853b-3e0c69dd6891.png)

Reducing the horizontal margin to 10px is OK, but not with fixedColumns.

![deploy-preview-20--alltheplaces netlify app_spiders html(Pixel 5)](https://user-images.githubusercontent.com/23022/185915414-9781c659-7c91-486f-8e60-b8cd3b42953c.png)

Would need to set a maximum width and force the text to wrap, perhaps.